x-pack/legacy/plugins/fleet/public/components/agent_details/agent_events_table.tsx

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
what do you think of moving these component files into a subdirectory into the agent details page?
` x-pack/legacy/plugins/fleet/public/pages/agent_details/components/[...]`

Reply:
Yes I like the idea